packages/fortinet_fortimanager/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -29,8 +29,8 @@ processors:
       field: temp
       tag: 'kv_temp'
       target_field: _temp
-      field_split: ( (?=(?:[^'"]|'[^']*'|"[^"]*")*$))
-      value_split: (?<!\\)=
+      field_split: ((?<=(?:[a-z0-9\"])) (?=(?:[a-z_]+=)))
+      value_split: =
       trim_value: "\""
```

While we are here...
```suggestion
      trim_value: '"'
```